### PR TITLE
DT-18470 Adjusting Yellow Ribbon GA tags

### PR DESCRIPTION
**Pull request by zacharymorel**
## Description
Adjusted tags to make YR consistent with other site wide searches. 

## Testing done
Ran browser locally and tested search. 

## Acceptance criteria
- [x] Change GA tags as reflected in this comment https://github.com/department-of-veterans-affairs/va.gov-team/issues/18470#issuecomment-794168123.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/yellow-ribbon/containers/SearchResults/index.jsx [hunk 1]
```diff
@@ -126,11 +126,11 @@ export class SearchResults extends Component {
       },
       'search-results-top-recommendation': undefined,
       'search-selection': 'Yellow Ribbon',
-      'search-result-chosen-page-url': window.location.href,
+      'search-result-chosen-page-url': school?.insturl || undefined,
```
Comment (kelsonic) [inline src/applications/yellow-ribbon/containers/SearchResults/index.jsx:129]: This is a business requirement change, just validating this is desired! 🙂

File: src/applications/yellow-ribbon/actions/index.js [hunk 2]
```diff
@@ -104,8 +104,8 @@ export const fetchResultsThunk = (options = {}) => async dispatch => {
           contributionAmount: contributionAmount || undefined,
           numberOfStudents: numberOfStudents || undefined,
         },
-        'search-total-results': response.results.length,
-        'search-total-result-pages': Math.ceil(
+        'search-results-total-count': response.results.length,
+        'search-results-total-page': Math.ceil(
```
Comment (kelsonic) [inline src/applications/yellow-ribbon/actions/index.js:108]: Did you let analytics know about these key changes? I'm guessing this is coming from their request anyways but just double checking! 🙂

**Review by kelsonic — APPROVED** [review]
LGTM ✅

File: src/applications/yellow-ribbon/actions/index.js (hunk 2 above)
Comment (zacharymorel) [reply src/applications/yellow-ribbon/actions/index.js:108]: Yep! It's an effort to make all va gov search's have the same skeleton of properties.

File: src/applications/yellow-ribbon/containers/SearchResults/index.jsx (hunk 1 above)
Comment (zacharymorel) [reply src/applications/yellow-ribbon/containers/SearchResults/index.jsx:129]: Yep! I'm used the incorrect URL when I first built the object.

**Pull request merged**